data/data/openstack/variables-openstack.tf

```diff
@@ -15,12 +15,6 @@ variable "openstack_base_image_name" {
   description = "Name of the base image to use for the nodes."
```

Comment:
We no longer need the `openstack_base_image_name` variable it seems.

Reply:
I was going to remove it too, but then I realized that we also need to support overridden image names (which are not created by the installer). It means that we have to to provide these names to Terraform somehow, and `openstack_base_image_name` is the only way to do it.

Reply:
Oh yeah, of course. I didn't notice it was still used on line 79 :+1: